_pytest/fixtures.py

```diff
@@ -1068,6 +1068,9 @@ def parsefactories(self, node_or_obj, nodeid=NOTSET, unittest=False):
         self._holderobjseen.add(holderobj)
         autousenames = []
         for name in dir(holderobj):
+            # Skip properties before the following getattr executes them.
+            if isinstance(getattr(type(holderobj), name, None), property):
```

Comment:
(Indeed the third argument to `type(...)` must be a dict, `None` or other types are not accepted)